package.json

```diff
@@ -921,6 +921,7 @@
     "@folio/plugin-find-user": "^6.0.0"
   },
   "resolutions": {
-    "moment": "~2.24.0"
+    "moment": "~2.24.0",
+    "@vue/compiler-sfc": "3.2.33"
```

Comment:
@Terala-Priyanka this should not be required after https://github.com/folio-org/ui-users/pull/2144 is merged

Reply:
@mkuklis 
Thank you for the inputs. Shall I put this PR in draft mode till #2144 is merge?

Reply:
@mkuklis I have removed the dependency resolution from package.json in the interest of #2144